microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -699,7 +708,7 @@
                       <table id="taxonomyTable" class="display" style="width:100%">
                         <thead>
                         <tr>
-                          <th>Relative Abundance</th>
+                          <th>Relative Abundance (Fraction of Sequences)</th>
```

Comment:
Would it be easy to transform this to "% of your sample", and round to 2 sig digits?